factory-containers/assemble-system-image

```diff
@@ -178,9 +213,15 @@ def copy_container_images_to_wic(target: dict, app_image_dir: str, wic_image: st
     # Therefore, in order to detect such subtle/silent error the following check is enabled,
     # it just throws an exception if the tar file does not exist prior to injecting it to the target wic file.
 
-    if not os.path.exists(app_image_tar_src):
+    try:
+        st = os.stat(app_image_tar_src)
```

Comment:
@mike-scott Do you think it's good enough for the juul case to increase the wic image enough to accommodate the tarball and on a first boot something (??) will increase rootfs further to accommodate the untarred image data? Or should we actually increase the wic image by the size of the untarred container image data?